content/en/apps/guides/integrations/rapidpro.md

```diff
@@ -166,3 +165,103 @@ Another common action you will likely need to perform in RapidPro is getting inf
 5. Use standards in logic where possible (ex. The non-zero value is true; 0=false, 1=true, 0=no, 1=yes)
 6. Sign the first message with the partner’s name, or MOH for visibility
 7. Consider providing an option to revisit information again (ex. Text 123 to this number to receive these messages again)
+
+### Lessons from integrations with the CHT
+
+The following are best practices when working on a RapidPro deployment to integrate with the CHT. Review RapidPro’s documentation to familiarize yourself with various components that include the [API](https://rapidpro.io/api/v2/). The next sections will focus on various components or stages in the deployment process.
```

Comment:
This can be largely eliminated, and the parts introducing RapidPro and it's API can be added to the top level intro paragraph.